### ISSUE TITLE
Include semver info (patch, minor, major) to Release Changelog

### ISSUE THREAD

**Issue by shawntabrizi** [paritytech/polkadot-sdk#6422]
### Is there an existing issue?

- [X] I have searched the existing issues

### Experiencing problems? Have you tried our Stack Exchange first?

- [X] This is not a support question.

### Motivation

When upgrading Polkadot SDK defendant projects to a new Polkadot SDK version, it is likely there are some breaking changes.

At the moment, existing change logs describe all the things that happened, but do not specify if those changes are actually breaking or not.

This leaves the end user needing to search through each change to determine which ones are breaking, and which ones are not.

### Request

We should include more metadata details from the pr docs into the changelog, specifically (but not limited to):

- crates affected
- semver details (patch, minor, major)

Ideally, we would also organize the changelog into "breaking changes" section, and then the rest. 

### Solution

update the scripts which generate the changelog to expose these details.

### Are you willing to help with this request?

Maybe (please elaborate above)

**Comment by ggwpez** [comment]
> Ideally, we would also organize the changelog into "breaking changes" section, and then the rest.

Yes this seems like a good improvement. Possibly we have to create multiple CHANGELOGs, one indexed by bump level and one by crate, or we somehow manage to squeeze all in one. cc @EgorPopelyaev

**Comment by bkchr** [comment]
IMO what we need is some sort of a more interactive changelog. To let people filter by whatever they need etc. Stuff like "what is breaking" is for example already expressed by the version bump you see.

**Comment by shawntabrizi** [comment]
We can probably make a simple UI easily against the PR doc folder: https://github.com/paritytech/polkadot-sdk/tree/master/prdoc

But what is the way to detect which PR doc are included in which release?

Why are some in versioned folders while most are not?

**Comment by bkchr** [comment]
AFAIK they are getting moved to the folders when they are part of the release. Not sure this maybe stopped? 

CC @EgorPopelyaev
